src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTaskBase.cs

```diff
@@ -295,15 +293,7 @@ public async Task<IOperationResult> Execute(Action<IOperationProgress> onProgres
 
                     // after the db for restore is done, we can safely set the db state to normal and write the DatabaseRecord
                     databaseRecord.DatabaseState = DatabaseStateStatus.Normal;
-                    var (updateIndex, _) = await _serverStore.WriteDatabaseRecordAsync(databaseName, databaseRecord, null, RaftIdGenerator.DontCareId, isRestore: true);
-                    await _serverStore.Cluster.WaitForIndexNotification(updateIndex);
-
-                    if (databaseRecord.Topology.RelevantFor(_serverStore.NodeTag))
```

Comment:
Why this was removed?

Reply:
It isn't needed anymore.
We don't wait for the database to be online in order to complete the restore operation.